web-app/prisma/migrations/20251024151044_share_refactoring/migration.sql

```diff
@@ -0,0 +1,46 @@
+/*
+  Warnings:
+
+  - You are about to drop the `share` table. If the table is not empty, all the data it contains will be lost.
+
+*/
+-- DropForeignKey
+ALTER TABLE "public"."share" DROP CONSTRAINT "share_creatorId_fkey";
+
+-- DropForeignKey
+ALTER TABLE "public"."share" DROP CONSTRAINT "share_jobId_fkey";
+
+-- DropForeignKey
+ALTER TABLE "public"."share" DROP CONSTRAINT "share_recipientOrganizationId_fkey";
+
+-- DropTable
+DROP TABLE "public"."share";
+
+-- CreateTable
+CREATE TABLE "jobShare" (
```

Comment:
I remember that we discuss about having jobShare shared with multiple users and orgs. like users[] and organizations[].

From what I see now we've linked it with users and org table. In this case we'll need to have "duplicated" shares for each user and org we want to. 

Did I got it right? if so that's desired outcome here?